src/minecraft_protocol.rs

```diff
@@ -139,6 +174,15 @@ impl<T: Write> MinecraftProtocolWriter for T {
         self.write_i32::<BigEndian>(v).unwrap();
     }
 
+    fn write_int_array(&mut self, v: Vec<i32>) {
+        v.iter()
+            .for_each(|element| self.write_i32::<BigEndian>(*element).unwrap());
```

Comment:
we could probably alter the packet macro to create this array type for everything automatically but eh 🤷‍♂ there aren't that many array types so this is fine